src/mavis/tools/vcf.py

```diff
@@ -203,6 +203,16 @@ def convert_record(record: VcfRecordType) -> List[Dict]:
                 }
             )
 
+        '''
+        As per VCF 4.2 specifications (https://samtools.github.io/hts-specs/VCFv4.2.pdf): 
```

Comment:
So this works but its a bit misleading. Technically start_position = 1 is only b/c we set that as default. It would make more sense to do this where the 0 is created (just after calling parse_bnd_alt)

https://github.com/bcgsc/mavis/blob/feature/sniffle-convert-handling/src/mavis/tools/vcf.py#L158

```python
            chr2, end, orient1, orient2, ref, alt = parse_bnd_alt(alt)
```

to

```python
            chr2, end, orient1, orient2, ref, alt = parse_bnd_alt(alt)
            if end == 0:
                end = 1  # telomeric BND alt syntax https://github.com/bcgsc/mavis/issues/294
```

Reply:
chatted offline, though rare, pos can also be 0